src/Button/Button.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import Button from "./";
+
+const meta = {
+  title: "Atoms/Button",
+  component: Button,
+} satisfies Meta<typeof Button>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Black: Story = {
+  args: {
+    children: "Black",
+    color: "black",
+    ariaLabel: "Example black button",
+    onClick: () => {},
```

Comment:
One of the features of Storybook that I want us to get into is using [actions](https://storybook.js.org/docs/essentials/actions) in the playground. It takes a bit of setup, though:

1. `npm i -D @storybook/addon-actions@8`
2. `import { action } from "@storybook/addon-actions";`
3. Change these `onClick` props to `action("click")`.

Once in place, we will get an inspector for every action logged:
![image](https://github.com/user-attachments/assets/f75f5e36-1ee1-4e14-90d1-16e2f18fd737)